src/pages/getting-started.js

```diff
@@ -1,18 +1,20 @@
-import GatsbyLink from "gatsby-link";
+import { Link as GatsbyLink } from "gatsby";
 import Helmet from "react-helmet";
 import React from "react";
 
 import Breadcrumbs from "../components/Breadcrumbs";
+import Layout from "../components/Layout";
 import Link from "../components/Link";
 import ListItem from "../components/ListItem";
 import Paragraph from "../components/Paragraph";
 import Title from "../components/Title";
 import UnorderedList from "../components/UnorderedList";
 
+import "../index.css";
```

Comment:
Having multiple `index.css` files being imported into a component is quite confusing. We should name the top level `src/index.css` something new to describe what value it brings.

Reply:
I think `app.css` would work here.

Reply:
@joelcarr I've renamed the file.